src/manifest.c

```diff
@@ -801,6 +823,17 @@ struct list *create_update_list(struct manifest *server)
 			}
 		}
 
+		/* Delta manifests don't include unchanged files in the FROM file list,
+		 * so files in the TO file list without a peer are not new when the file
+		 * exists on the system. */
+		filename = mk_full_filename(path_prefix, file->filename);
+		ret = verify_file_lazy(filename);
```

Comment:
we talked about just using the TO file list as "new or changed" files and combining that metric into one report. Is that feasible to do?

Reply:
We can already count changed files because delta manifests will have entries for them. The problem here is that unchanged files will not be tracked in the FROM file list when using delta manifests. As a result, the old method could not differentiate unchanged files from new ones.

I don't think combining unchanged and new files would be useful.